### PR TITLE
Some small fixes and changes

### PR DESCRIPTION
**Pull request by dl1jbe**

### CODE REVIEW

File: src/readcalls.c [hunk 1]
```diff
@@ -206,7 +206,6 @@ void count_contest_bands(int check, int *count) {
 int readcalls(void) {
```
Comment (zcsahok) [inline src/readcalls.c:206]: in the future we could consider using `parse_logline()` in  this function (from `writecabrillo.c`)

**Review by zcsahok — APPROVED** [review]
I'll merge these and the previous changes to dev_plugin to keep the two branches in sync, if that's ok.

File: src/readcalls.c (hunk 1 above)
Comment (dl1jbe) [reply src/readcalls.c:206]: I am working in that direction. 
The used result struct is not quite ideal as it mixes parse results from loglines and QTC lines. Maybe I can decouple it.

Next step will be better handling of worked stations (collecting all infos about a call in the worked_t stuct). After adding some tests to readcalls() I want to use the parsed struct for addcall and addmult.

**Comment by dl1jbe** [comment]
> I'll merge these and the previous changes to dev_plugin to keep the two branches in sync, if that's ok.

Fine, good idea.

**Pull request merged**